WIP: Update Documentation Data

arianelestrade opened the pull request:
Cette pull request a pour objet le fait de créer un folder bdd_documentation qui recensera : 
- une description de la base de données à actualiser
- un fichier pdf qui complète cette description pour les données Diane
- une description des variables pour le nouveau modèle (sans variables DIANE).

arianelestrade committed:
creation du fichier liste_var_pour_dgfip

arianelestrade committed:
move_description-donnees

Move description-donnees.md to folder description_donnees

vviers commented:
Super ! :)

Petite suggestion :
    - renommer le dossier ~`description_bdd`~ -> `data_catalogue`
    - ~mettre le fichier `description_donnees.md` dedans~ my bad c'est déjà fait 😅 

Je passe la PR en draft, tu peux la mettre en "Ready to merge" quand tu as tout poussé dedans :)

arianelestrade committed:
import_doc_diane

Import doc diane to have more detailed information on diane variables described in description-donnees.md

arianelestrade committed:
update_liste_var_pour_dgfip

Listing jusqu'au données sur les cotisations sociales et débits.

vviers commented on description_bdd/liste_var_pour_dgfip:
```diff
@@ -0,0 +1 @@
+
```
Le repo contient de la doc à destination du grand "public" donc je serais plutôt pour appeler ça `liste_vars_nouveau_modele.md`

arianelestrade committed:
correct_names

arianelestrade committed:
add_cotisation

Rajout et update des variables sur les cotisations sociales.

arianelestrade committed:
fin_add_variables

Ajout des dernières variables (délai et procédure collective), update des variables sur les cotisations sociales.

arianelestrade committed:
minor_correction

arianelestrade committed:
minor_correction

arianelestrade commented:
J'ai fini les modifs! Je n'arrive pas à modifier le folder depuis github, mais si tu veux renommer, n'hésite pas à le faire ! 

Dans données sur les cotisations sociales, j'ai mis les variables suivantes indiquée dans la grosse doc : Num_Ecn, Num_Hist_Ecn, Dt_trt_ecn . Mais je ne suis pas sûre de bien les comprendre, donc j'hésite à enlever... 

J'ai très peu d'info sur les variables initiales de la bdd urssaf sur les entrées en procédures collectives). Donc si tu as des choses à ajouter n'hésite pas. Pour la variable outcome, c'est repris de ce que m'avait dit Pierre, s'il y a moyen de re-vérifier dans le code ce serait cool !

arianelestrade committed:
minor_correction

Correctify variable "delai"

vviers commented on description_bdd/liste_var_input_modele.md:
```diff
@@ -0,0 +1,188 @@
+Les données utilisées proviennent de plusieurs sources:
+
+- **Données Sirene** Raison sociale, adresse, code APE, date de création etc.\
+- **Données Altarès** Données de défaillance, permet la définition de l'objectif d'apprentissage
```
Altarès c'est juste le paydex maintenant

> Nous avons utilisé les données fournies par Altares concernant les défaillances en Bourgogne Franche Comté (prestation payante). Comme cette base n'est pas disponible dans toutes les régions, ce seront les données de procédure collective fournies par l'URSSAF qui seront dorénavent utilisées.

arianelestrade committed:
add_new_var_althares

Ajoute un paragraphe sur les nouvelles données altharès concernant le retard de paiement envers les fournisseurs.

vviers commented on description_bdd/liste_var_input_modele.md:
données sur les délais de cotisation sociales

vviers reviewed (commented):
LGTM :)

Par contre, je vois que l'intégration continue de GitHub casse... Il faudrait regarder ce qu'il se passe mais je pense que les fichiers sont mal formatés.

raphaelventura commented:
Fermée, obsolète après mise à jour récente de la doc.
